src/Twilio/Http/SystemNetHttpClient.cs

```diff
@@ -36,62 +32,41 @@ public SystemNetHttpClient(System.Net.Http.HttpClient httpClient = null)
         /// <returns>Twilio response</returns>
         public override Response MakeRequest(Request request)
         {
-            var task = MakeRequestAysnc(request);
-            task.Wait();
-            return task.Result;
+            try
+            {
+                var task = MakeRequestAsync(request);
+                task.Wait();
+                return task.Result;
+            }
+            catch (AggregateException ae)
+            {
+                // Combine nested AggregateExceptions
+                ae = ae.Flatten();
+                throw (ae.InnerExceptions.Count == 1) ? ae.InnerException : ae;
```

Comment:
IMO we should always throw the first exception. We know the thing we're calling will only throw one exception so feels like a safe assumption.